semantics/language/common/expr/eval.k

```diff
@@ -172,14 +172,14 @@ module C-COMMON-EXPR-EVAL
 
      syntax Error ::= "errorLocInvalid"
                     | "errorLocBounds"
-     rule (.K => UNDEF("CEE2",
+     rule (.K => UNDEF("CEE4",
```

Comment:
So what is the rule for these codes?  I would explain it briefly in the spreadsheet with all the errors.